tests/LinterConfigTest.hack

```diff
@@ -108,18 +108,6 @@ final class LinterConfigTest extends HackTest {
     expect($config)->toBeNull('No config supplied');
   }
 
-  public function testConfigSuppliedAndLinterDoesNotSpecify(): void {
-    $lrc = static::getLintRunConfig();
-
-    $config = $lrc->getLinterConfigForLinter<FinalOrAbstractClassLinter, _>(
-      FinalOrAbstractClassLinter::class,
-    );
-
-    expect($config)->toEqual(
-      shape('I can supply a config' => "even if the linter doesn't need one"),
-    );
-  }
-
```

Comment:
@lexidor  any thoughts on this change? The main alternative is to specify all linters as taking `shape(...)` but that removes a lot of the safety.

Reply:
When I originally wrote this, I imagined that many linters that were not configurable before might want to become configurable. I wanted adding a configuration to be a forward compatible change. This didn't really happen, so it would be safe to remove this.